camera_info:
  fx: 911.6033325195312
  fy: 910.8851318359375
  cx: 655.0755615234375
  cy: 363.9165954589844